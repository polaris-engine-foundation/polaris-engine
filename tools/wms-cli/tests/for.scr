func main() {
    a[0] = 0;
    a[1] = 1;
    a[2] = 2;

    sum = 0;
    for(k, v in a) {
        print("k=" + k);
        print("v=" + v);
	sum = sum + v;
    }
    print("sum = " + sum);

    sum = 0;
    for(v in a) {
        sum = sum + v;
    }
    print("sum = " + sum);

    for(i in 0..9) {
        print("i=" + i);
    }
}
