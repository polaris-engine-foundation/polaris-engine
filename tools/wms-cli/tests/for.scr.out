k=2
v=2
k=1
v=1
k=0
v=0
sum = 3
sum = 3
i=0
i=1
i=2
i=3
i=4
i=5
i=6
i=7
i=8
i=9
